twitch.go

```diff
@@ -178,6 +181,24 @@ func scrolloReward(params string) {
 	f.WriteString(fmt.Sprintf(" %.256s ✨✨✨ ", params))
 }
 
+func playYoutubeReward(params string) {
+	var isValidID = regexp.MustCompile(`[a-zA-Z0-9_-]`).MatchString
+	url_args := strings.Split(params, "v=")[1] // Get the video ID arg from the rest of the URL
+	video_id := strings.Split(url_args, "&")[0] // Split out any other args from the URL, keeping only the video ID
+
+	if !(len(video_id)==11) {
```

Comment:
supernit: why not?
```len(video_id) != 11```
it just reads a bit clearer!

Reply:
Huh, I didn't know you could to this in Go. Thanks!